streams/src/test/java/org/apache/kafka/streams/processor/internals/AssignedTasksTest.java

```diff
@@ -192,7 +192,7 @@ public void shouldCloseTaskOnSuspendWhenRuntimeException() {
         mockInitializedTask();
         t1.suspend();
         EasyMock.expectLastCall().andThrow(new RuntimeException("KABOOM!"));
-        t1.close(false);
+        t1.close(false, false);
         EasyMock.expectLastCall();
         EasyMock.replay(t1);
 
```

Comment:
super nit: not part of this PR proper, but maybe set `RuntimeException` returned from the `suspendTask()` call to variable then use it in the `assert` statement? Makes things a little easier to understand.